### PR TITLE
Add MT script

### PR DESCRIPTION
**Pull request by Manonedde**
Script to compute MT maps.
Check code and test. 
Data PR (HoG subject) : https://drive.google.com/file/d/1qxzVORKmFPRa5ZXmxPTaR8KIx-Pbla4y/view?usp=sharing

### CODE REVIEW

**Review by arnaudbore — APPROVED** [review]

File: scripts/scil_compute_MT_maps.py [hunk 1]
```diff
@@ -0,0 +1,376 @@
...
+
+from scilpy.io.image import get_data_as_mask
+from scilpy.io.utils import (add_overwrite_arg, assert_inputs_exist,
+                             assert_output_dirs_exist_and_empty)
+
+EPILOG = """
+Helms G, Dathe H, Kallenberg K, Dechent P. High-resolution maps of
+magnetization transfer with inherent correction for RF inhomogeneity
+and T1 relaxation obtained from 3D FLASH MRI. Magnetic Resonance in Medicine.
+2008;60(6):1396‑407.
```
Comment (GuillaumeTh) [inline scripts/scil_compute_MT_maps.py:72]: @Manonedde could you please remove the - in `1396‑407` and readd it. You copy paste directly the URL and this character is not the real `-`.

File: scripts/scil_compute_MT_maps.py [hunk 2]
```diff
@@ -0,0 +1,376 @@
+# -*- coding: utf-8 -*-
```
Comment (GuillaumeTh) [inline scripts/scil_compute_MT_maps.py:1]: #!/usr/bin/env python3

**Review by GuillaumeTh — CHANGES_REQUESTED** [review]

File: scripts/scil_compute_MT_maps.py [hunk 3]
```diff
@@ -0,0 +1,377 @@
...
+    p.add_argument('out_dir',
+                   help='Path to output folder.')
+    p.add_argument('in_mask',
+                   help='Path to the T1 binary brain mask. Must be the sum '
+                        'of the three tissue probability maps from '
+                        'T1 segmentation (GM+WM+CSF).')
+    p.add_argument('--out_prefix',
+                   help='Prefix to be used for each output image.')
+    p.add_argument('--in_B1_map', action='store_true',
+                   help='Path to B1 coregister map to MT contrasts.')
```
Comment (mdesco) [inline scripts/scil_compute_MT_maps.py:89]: this option does not work. Please correct. If I remove the action='store_true', it works.

**Review by mdesco — APPROVED** [review]
Thanks Manon for this.

To me, ready to go! Good job!

**Pull request merged**